tests/structured_tests/split_1.toml

```diff
@@ -0,0 +1,9 @@
+query = """
+* | split on " "
+"""
+input = """
+Oct 09 20:22:21 web-001 influxd[188053]: 127.0.0.1 "POST /write \"escaped\" HTTP/1.0" 204
```

Comment:
`"""` is multiline but not raw -- for raw you want `'''` -- this string is actually `POST /write "escaped" HTTP/1.0"`

Reply:
Ah, that makes more sense - done!